assets/src/setup/components/nav/test/__snapshots__/index.js.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Would the Jest snapshots be good candidates to add to `.gitattributes` as `linguist-generated=true` so they are suppressed from being expanded in diff views?

Reply:
Good idea!